### PR TITLE
Add Valid IBAN Generator Test Helper

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

- So we don't have to change the IBANs all the time added a test helper class to randomly generate french IBANs
  - Resources: [Gateway helper functions for IBAN generation](https://github.braintreeps.com/braintree/gateway/pull/24498/files) and a [article about calculating values for IBAN generation](https://community.appway.com/screen/kb/article/generating-and-validating-an-iban-1683400256881)
  - Opened a new PR because the old one #910 was off of the `next-major-version` branch and not `5.x`

### Checklist

- [x] Added a changelog entry

- @jaxdesmarais

### CODE REVIEW

File: Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/// Provides helper functions to test the SEPA Direct Debit flow in sandbox.
+@objcMembers public class BTSEPADirectDebitTestHelper: NSObject {
+
+    /// Generates a valid 27-digit IBAN (International Bank Account Number) for testing SEPA Direct Debit flows.
+    /// - Returns: a valid IBAN
+    public static func generateValidIBAN() -> String {
```
Comment (scannillo) [inline Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift:8]: Can a merchant use this for prod testing? If not, we can specify only sandbox.

File: Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/// Provides helper functions to test the SEPA Direct Debit flow in sandbox.
```
Comment (scannillo) [inline Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift:3]: Same comment as last PR regarding - do we want to call out that this isn't under version control?

**Review by scannillo — APPROVED** [review]
A few nitpicks - but changes look great to me! This really helps clean up our testing story for merchants 👏

**Review by sshropshire — APPROVED** [review]
🌊

File: Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift (hunk 1 above)
Comment (jaxdesmarais) [reply Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift:8]: Good call, this is sandbox only - will add sandbox to the method signature!

File: Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift (hunk 2 above)
Comment (jaxdesmarais) [reply Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift:3]: Added: https://github.com/braintree/braintree_ios/pull/913/commits/6fe0c36815e5aa7cc1ab531c0f0d57a650151664

**Pull request merged**